AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminOpenCMSMediaGalleryPopupActionGroup.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminOpenCMSMediaGalleryPopupActionGroup">
+        <click selector="{{TinyMCESection.InsertImageIcon}}" stepKey="clickInsertImage"/>
```

Comment:
hi @eduard13 This is not correct buttons, as  WYSIWYG Editor must be disabled.
https://devdocs.magento.com/mftf/docs/getting-started.html
In the WYSIWYG Options section set the Enable WYSIWYG Editor option to Disabled Completely.